blocks/subscriptions-block/components/ContactInfo/FacebookIdentityIcon.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const FacebookIdentityIcon = () => (
```

Comment:
Any reason this icon and the Google icon are not placed in the SDK along with the other icons?

The icons if not moved to the SDK will need to have accessibility tags added to them. Are they images or just presentational?

Reply:
I think these are one offs at this point.  But did through the svg into an img and added an alt and title attributes